### PR TITLE
Fix installation instructions

### PR DESCRIPTION
**Pull request by microstudi**
Som suggestion to improve install instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ This is the repository for the [Decidim Documentation website](https://docs.deci
 [Install antora](https://docs.antora.org/antora/2.3/install-and-run-quickstart/) with software dependencies.
 
 ```bash
-# Install Node & npm using nvm
+# Install Node an Npm, for instance using nvm:
```
Comment (andreslucena) [inline README.md:15]:
```suggestion
# Install Node and Npm, for instance using nvm:
```

File: README.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ This is the repository for the [Decidim Documentation website](https://docs.deci
...
 wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
 nvm install node
 
-# Install Antora and dependencies
-npm install
+# Install Antora and dependencies (some system will need to prefix `sudo`)
+npm install -g antora
```
Comment (andreslucena) [inline README.md:20]: There are some other [packages](https://github.com/decidim/documentation/blob/master/package.json) for this to work as the production instance. Does the installation not work with `npm install`? Do you have a traceback or something?

**Review by andreslucena — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (microstudi) [reply README.md:20]: As a matter of fact, maybe is not necessary to have installed antora globally. In any case npm install will only work if it is executed inside the documention folder, which at this step is not cloned yet.

Comment (andreslucena) [reply README.md:20]: Wow, I hadn't noticed that. You're absolutely right, impossible that this works as it is now.

Comment (andreslucena) [reply README.md:20]: Can you move this to after line 26 ( `cd documentation/`) and leave it as `npm install` 🙏🏽?

Comment (microstudi) [reply README.md]: done!

**Pull request merged**